app/assets/stylesheets/components/comments.scss

```diff
@@ -0,0 +1,31 @@
+@import '../config/import';
+
+.crayons-comment {
+  margin-bottom: var(--su-4);
+  font-size: var(--fs-s);
+
+  &__meta {
+    display: flex;
+    align-items: center;
+
+    a {
+      display: flex;
+      align-items: center;
+    }
+  }
+
+  &__body {
+    padding-left: var(--su-7);
+
+    a:hover {
+      text-decoration: underline; //todo: default for all in-content links
```

Comment:
Is this a TODO for this PR @ludwiczakpawel or @joshpuetz ?

Reply:
It's not mine.. @ludwiczakpawel ?

Reply:
not for this PR - general todo for when we'll be tackling some links styling. happy to remove it if the comment is an issue.